lmdeploy/tokenizer.py

```diff
@@ -231,6 +231,8 @@ def _check_transformers_version(self, model_dir: str):
 
         current_transformers_version = version.parse(transformers.__version__)
         cfg = get_model_arch(model_dir)[1]
+        if getattr(cfg, 'transformers_version', 'str') is None:
+            cfg = getattr(cfg, 'llm_config')
         required_transformers_version = version.parse(
```

Comment:
Do I need to change this line to these?
```python3
        cfg_ver = getattr(cfg, 'transformers_version', 'str')
        if cfg_ver is None:
            return
        required_transformers_version = version.parse(cfg_ver)
```

Reply:
yes, should be `None` instead of `'str'`

Reply:
ok

Reply:
done cc @RunningLeon